src/config_templates/config/gearman.php.tpl

```diff
@@ -0,0 +1,14 @@
+<?php
+
+{{STATIC_FILE_HEADER}}
+
+return [
+    'gearman' => [
+        'address'    => '{{address}}',
+        'port'       => {{port}},
+        'encryption' => {{encryption}},
+        'password'   => '{{password}}',
```

Comment:
The `encryption` and `password` options are not used with V4 anymore and should be removed from the new config template.